src/fluree/db/peer/http_api.clj

```diff
@@ -112,9 +112,10 @@
 
 (defn decode-body
   [body type]
-  (case type
-    :json (let [^bytes body' (.bytes ^BytesInputStream body)]
-            (-> body' (String. "UTF-8") json/parse))))
+  (let [body' (-> ^BytesInputStream body .bytes (String. "UTF-8"))]
```

Comment:
This might cause reflection w/o the `^bytes` type hint. Did you run `make eastwood` on it?

Reply:
It passed the `make ci` without any issues.

I'll test with the following:
```
(defn decode-body
  [^bytes body type]
  (let [body' (-> ^BytesInputStream body .bytes (String. "UTF-8"))]
    (case type
      :string body'
      :json   (json/parse body'))))
```
^^^
That passes linting as well

Reply:
If the original code doesn't bother the linter then it's 👍🏻